lcls-twincat-optics/lcls_twincat_optics_plc/POUs/Helpers/MC_SmoothMover.TcPOU

```diff
@@ -0,0 +1,78 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4020.12">
+  <POU Name="MC_SmoothMover" Id="{49439872-0161-4fd4-9e7f-62bcb9508941}" SpecialFunc="None">
```

Comment:
This is another FB that needs to become part of the motion lib default behavior at some point. E.g. updating the `fPosition` variable during an active move should transition to the new setpoint.